pkg/aggregator/mocksender/asserts.go

```diff
@@ -43,29 +48,33 @@ func (m *MockSender) AssertMetricNotTaggedWith(t *testing.T, method string, metr
 	return m.Mock.AssertCalled(t, method, metric, mock.AnythingOfType("float64"), mock.AnythingOfType("string"), AssertTagsNotContains(t, tags))
 }
 
+// Return a bool value if all the elt of expected are inside the actual array
+func expectedInActual(actual, expected []string) bool {
+	expectedCount := 0
```

Comment:
You can avoid creating that variable thanks to [loop labels](http://relistan.com/continue-statement-with-labels-in-go/):
```
if e == a {
  continue ITER_EXPECTED
}
return false
```

Reply:
@xvello I'm not sure it's a good idea to use labels, maybe you'll avoid creating a variable but IMHO you'll lose much more in code readability. (it's not indented + not many people use this syntax, and opens the `goto` door 😬)

Reply:
The implementation looks like that:
```ITER:
	for _, e := range expected {
		for _, a := range actual {
			if e == a {
				continue ITER
			}
		}
		return false
	}
	return true
```
I prefer creating a variable it's more explicit and a common way to implement this kind of search loop

Reply:
okay then ;)